packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeUtils.ts

```diff
@@ -69,10 +69,20 @@ export class EscapedPath {
     }
 }
 
+export interface ISummaryNode {
```

Comment:
Curious why add the interface?

Reply:
Never mind, I think I get it now. During `completeSummary`, you will replace the pending summary with an extended (GC) version of it with the used state.

I don't think you _need_ the interface still, because `SummaryNodeWithGC` should satisfy type constraint `SummaryNode`.

Reply:
You are right. The interface is not needed. I was running into some weird error because of which I created the interface but I don't remember what it was and don't see the error anymore! Removed it.